cmd/ci-operator/main.go

```diff
@@ -279,7 +279,7 @@ func setupLogger() (*secrets.DynamicCensor, io.Closer, error) {
 	if !set {
 		return &censor, nil, nil
 	}
-	verboseFile, err := os.Open(filepath.Join(artifactDir, "ci-operator.log"))
+	verboseFile, err := os.Create(filepath.Join(artifactDir, "ci-operator.log"))
```

Comment:
What if it already exists?

Reply:
Well, it clearly doesn't :)

Reply:
But how did it go through e2e then?

Reply:
So `Create` does not fail when the file exists, it just truncates it. I think this is fine behavior. Added code here to make the dir if it does not exist, perhaps that is the cause here. Not entirely sure, but we should be OK to merge the safer code ...